ownCloud/SDK Extensions/OCItem+Extension.swift

```diff
@@ -216,4 +216,21 @@ extension OCItem {
 	func fileSuffix() -> String {
 		return (self.name as NSString).pathExtension
 	}
+
+	var sizeInReadableFormat: String {
+		let byteCountFormatter = ByteCountFormatter()
+		byteCountFormatter.countStyle = .file
+		let size = byteCountFormatter.string(fromByteCount: Int64(self.size))
+		return size
+	}
+
+	var lastModifiedInReadableFormat: String {
```

Comment:
@pablocarmu Creating formatter instances on the fly is very expensive. I would propose having those lazily created and cached statically, so that they can be re-used.

Reply:
yeah, make sense. I'll change it and push it!